bigtable/tests/unit/test_row.py

```diff
@@ -330,35 +335,56 @@ def test_delete_cells_with_string_columns(self):
                 column_qualifier=column2_bytes,
             ),
         )
-        self.assertEqual(row._pb_mutations, [expected_pb1, expected_pb2])
+        self.assertEqual(row.row_mutations.mutations_entry.mutations[0],
+                         expected_pb1)
+        self.assertEqual(row.row_mutations.mutations_entry.mutations[1],
+                         expected_pb2)
+
+    def _make_responses(self, codes):
+        import six
+        from google.cloud.bigtable_v2.proto.bigtable_pb2 import (
+            MutateRowsResponse)
+        from google.rpc.status_pb2 import Status
+
+        entries = [MutateRowsResponse.Entry(
+            index=i, status=Status(code=codes[i]))
+            for i in six.moves.xrange(len(codes))]
+        return MutateRowsResponse(entries=entries)
 
     def test_commit(self):
-        from google.protobuf import empty_pb2
+        from grpc import StatusCode
         from google.cloud.bigtable_v2.gapic import bigtable_client
+        from google.cloud.bigtable_admin_v2.gapic import (
+            bigtable_table_admin_client)
 
         project_id = 'project-id'
+        instance_id = 'instance-id'
+        table_id = 'table-id'
         row_key = b'row_key'
-        table_name = 'projects/more-stuff'
         column_family_id = u'column_family_id'
         column = b'column'
 
-        api = bigtable_client.BigtableClient(mock.Mock())
+        data_api = bigtable_client.BigtableClient(mock.Mock())
+        table_api = bigtable_table_admin_client.BigtableTableAdminClient(
+            mock.Mock())
         credentials = _make_credentials()
         client = self._make_client(project=project_id,
                                    credentials=credentials, admin=True)
-        table = _Table(table_name, client=client)
+        instance = client.instance(instance_id)
+        table = instance.table(table_id)
```

Comment:
Why are we creating "real" instance / table objects here?  We stub out the `table_data_client` below, which is all that the `row.commit` call needs.

Reply:
@tseaver Here I am getting instance / table from mocked client which is on self._make_client().